docs/guides/jupyter.rst

```diff
@@ -0,0 +1,398 @@
+
+   This is a example of a Jupyter notebook stored in MyST Markdown format.
+
+   ```{code-cell} ipython3
+   import sys
+   print(sys.version)
+   ```
+
+   ```{code-cell} ipython3
+   from IPython.display import Image
+   ```
+
+   ```{code-cell} ipython3
+   Image("http://sipi.usc.edu/database/preview/misc/4.2.03.png")
+   ```
+
+To render this notebook in Sphinx using nbsphinx and jupytext,
+you will need to add this to your ``conf.py``:
+
+.. tabs::
+
+   .. tab:: nbsphinx
+
+      .. code-block:: python
+         :caption: conf.py
+
+         nbsphinx_custom_formats = {
+             ".md": ["jupytext.reads", {"fmt": "mystnb"}],
+         }
+
+   .. tab:: MyST-NB
+
+      .. code-block:: python
+         :caption: conf.py
+
+         nb_custom_formats = {
+             ".md": ["jupytext.reads", {"fmt": "mystnb"}],
+         }
+
+Notice that the Markdown format does not store the outputs of the computation.
+nbsphinx will automatically execute notebooks without outputs,
+so in your HTML documentation they appear as complete.
+
+.. _notebook-galleries:
+
+Creating galleries of examples using notebooks
+----------------------------------------------
+
+`nbsphinx`_ has support for :doc:`creating thumbnail galleries from a list of Jupyter
+notebooks <nbsphinx:subdir/gallery>`.
+This functionality relies on `Sphinx-Gallery <https://sphinx-gallery.github.io/>`_
+and extends it to work with Jupyter notebooks rather than Python scripts.
+
+To use it, you will need to install both nbsphinx and Sphinx-Gallery,
+and modify your ``conf.py`` as follows:
+
+.. code-block:: python
+   :caption: conf.py
+
+   extensions = [
+      'nbsphinx',
+      'sphinx_gallery.load_style',
+   ]
+
+After doing that, there are two ways to create the gallery:
+
+- From a reStructuredText source file, using the ``.. nbgallery::`` directive,
+  :ref:`as showcased in the
+  documentation <nbsphinx:/a-normal-rst-file.rst#thumbnail-galleries>`.
+- From a Jupyter notebook, adding a ``"nbsphinx-gallery"`` tag to the metadata of a cell.
+  Each editor has a different way of modifying the cell metadata (see figure below).
+
+.. figure:: /_static/images/guides/jupyterlab-metadata.png
+   :width: 80%
+   :align: center
+   :alt: Panel to modify cell metadata in JupyterLab
+
+   Panel to modify cell metadata in JupyterLab
+
+For example, this reST markup would create a thumbnail gallery
+with generic images as thumbnails,
+thanks to the Sphinx-Gallery default style:
+
+.. code-block:: rest
+
+   Thumbnails gallery
+   ==================
+
+   .. nbgallery::
+      notebooks/Example 1
+      notebooks/Example 2
+
+.. figure:: /_static/images/guides/thumbnail-gallery.png
+   :width: 80%
+   :align: center
+   :alt: Simple thumbnail gallery created using nbsphinx
+
+   Simple thumbnail gallery created using nbsphinx
+
+To see some examples of notebook galleries in the wild:
+
+- `poliastro <https://docs.poliastro.space/>`_ offers tools for interactive Astrodynamics in Python,
```

Comment:
It seems that there some bullets missing from this list. I was expecting to see more examples but there is just one project listed here. I think we should add more bullets or remove the listing and left only one example.

Reply:
Done!